lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -967,12 +968,16 @@ def set_http_request(record, entry)
 
       latency = input.delete('latency')
       unless latency.nil?
-        # Parse latency to duration. If failed, skip setting latency.
-        duration_in_seconds = ChronicDuration.parse(latency)
-        if duration_in_seconds
+        # Parse latency. If no valid format is detected, skip setting latency.
+        # Format: space/tab (optional) + integer + point & decimal (optional)
+        #       + space/tab (optional) + "s" + space/tab (optional)
+        # e.g.: "1.42 s"
+        parsed_groups = latency.scan(LATENCY_REGEX)
+        if !parsed_groups.empty? && parsed_groups[0].length == 2
```

Comment:
Yeah, it's kind of redundant. I just removed it. I was mainly worried what if someone mistakenly touches the regex and the length changes? It might crash the agent if the tests failed to catch it. Yet since we've changed the regex to have the groups named as `seconds` and `decimal`, I think the chance of such mistake should be quite low. I'll stop worrying about it.